From 0f0a4b54142d8b424969f0b6ace6fc8b53b1d05d Mon Sep 17 00:00:00 2001
From: Jan Beich <jbeich@FreeBSD.org>
Date: Mon, 24 Jan 2022 12:32:00 +0000
Subject: [PATCH] meson: Drop unused argument for i18n.merge_file()

Ignored in Meson < 0.60.0, deprecated since 0.60.1 and fatal since 0.61.0.

data/appdata/meson.build:2:0: ERROR: Function does not take positional arguments.
data/meson.build:5:0: ERROR: Function does not take positional arguments.

Part-of: <https://gitlab.gnome.org/GNOME/polari/-/merge_requests/242>
---
 data/appdata/meson.build | 2 +-
 data/meson.build         | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/appdata/meson.build b/data/appdata/meson.build
index 88b4a0be..781ba22f 100644
--- a/data/appdata/meson.build
+++ b/data/appdata/meson.build
@@ -1,5 +1,5 @@
 appdata_name = app_id + '.appdata.xml'
-appdata = i18n.merge_file(appdata_name,
+appdata = i18n.merge_file(
   input: appdata_name + '.in',
   output: appdata_name,
   po_dir: '../../po',
diff --git a/data/meson.build b/data/meson.build
index cedfd57b..655ae700 100644
--- a/data/meson.build
+++ b/data/meson.build
@@ -2,7 +2,7 @@ subdir('appdata')
 subdir('icons')
 
 desktop_filename = app_id + '.desktop'
-desktop_file = i18n.merge_file(desktop_filename,
+desktop_file = i18n.merge_file(
   input: desktop_filename + '.in',
   output: desktop_filename,
   po_dir: '../po',
-- 
GitLab

